associate-software-engineer/fc-1/javascript/modulo-3/topico-2/ex-22.js

```diff
@@ -0,0 +1,35 @@
+const consumo = {
+    "estudantes": [
+      {"id": 1001, "dataNascimento": "2005-03-02"},
+      {"id": 1002, "dataNascimento": "2003-07-15"},
+      {"id": 1003, "dataNascimento": "2004-11-21"},
+      {"id": 1004, "dataNascimento": "2006-01-04"},
+      {"id": 1005, "dataNascimento": "2002-05-09"}
+    ],
+    "compras": [
+      {"idEstudante": 1001, "categoria": "Tecnologia", "valor": 500, "dataCompra": "2023-03-15"},
+      {"idEstudante": 1002, "categoria": "Livros", "valor": 80, "dataCompra": "2023-03-20"},
+      {"idEstudante": 1003, "categoria": "Roupas", "valor": 200, "dataCompra": "2023-03-22"},
+      {"idEstudante": 1004, "categoria": "Jogos", "valor": 150, "dataCompra": "2023-03-25"},
+      {"idEstudante": 1005, "categoria": "Tecnologia", "valor": 1200, "dataCompra": "2023-03-28"},
+      {"idEstudante": 1001, "categoria": "Roupas", "valor": 300, "dataCompra": "2023-03-30"},
+      {"idEstudante": 1003, "categoria": "Tecnologia", "valor": 750, "dataCompra": "2023-04-02"},
+      {"idEstudante": 1002, "categoria": "Alimentação", "valor": 50, "dataCompra": "2023-04-05"},
+      {"idEstudante": 1004, "categoria": "Livros", "valor": 90, "dataCompra": "2023-04-07"},
+      {"idEstudante": 1005, "categoria": "Jogos", "valor": 400, "dataCompra": "2023-04-10"}
+    ]
+}
+
+function calculateAge(dateOfBirth) {
+  const currentDate = new Date();
+  const birthDate = new Date(dateOfBirth);
+  const ageInMilliseconds = currentDate - birthDate;
+  const ageInYears = ageInMilliseconds / 1000 / 60 / 60 / 24 / 365;
+  return Math.round(ageInYears);
+}
+
+consumo.estudantes.forEach(person => {
```

Comment:
esqueceu o segundo exercício que é classificar nos ranges e trazer as compras lembra?

Reply:
to fazendo ainda hehe